keps/sig-api-machinery/1904-efficient-watch-resumption/README.md

```diff
@@ -300,8 +300,8 @@ Alpha should provide basic functionality covered with tests described above.
 #### Beta -> GA Graduation
 
 - Enabled in Beta for at least two releases without complaints
-- Rolling-upgrade of kube-apiservers in 5k-node cluster test is
-   automated and running periodically.
+- Benefits of the feature during rolling upgrade of kube-apiserver at scale
+   (thousands of nodes) proven in production
```

Comment:
We still aren't testing HA clusters in upstream. Given that it will take more time (we're still trying to converge on what exact framework we should be using) and the fact that:
- we're somewhat-continuously testing it as prt of GKE
- we've already seen a proven effect/benefit from it on real production clusters

I don't want to block GA-ing this feature on that.